lib/dotcom_web/components/system_status/status_row_heading.ex

```diff
@@ -48,12 +65,45 @@ defmodule DotcomWeb.Components.SystemStatus.StatusRowHeading do
       <.subway_route_pill class="group-hover/row:ring-brand-primary-lightest" route_ids={@route_ids} />
     </div>
 
-    <div class="pr-2 flex items-center">
+    <div class="h-6 pr-2 flex items-center">
       <.status_icon status={@status} />
     </div>
 
-    <div class="grow flex items-center">
+    <div class="grow flex flex-wrap items-baseline gap-x-2">
       <.status_label status={@status} prefix={@prefix} plural={@plural} />
+      <.subheading text={@subheading_text} />
+    </div>
+    """
+  end
+
+  defp subheading_text(%{status: :station_closure, alerts: alerts, route_ids: route_ids}) do
+    affected_stops = @affected_stops.affected_stops(alerts, route_ids)
```

Comment:
Having this call here feels _weird_ to me, because we're now making this call out to `@affected_stops`, which makes calls out to some of the repo's, from a component.

The alternative was to duplicate this call across the three touchpoints that currently use it, which I did try, but I wasn't thrilled with how cluttered the result was. Open to other ideas here, either to implement now or as follow-ups!